### ISSUE TITLE
Partially transient input: backward fill in time?

### ISSUE THREAD

**Issue by Manangka** [Deltares/imod-python#86]
In GitLab by @Huite on Aug 22, 2019, 14:34

I've come across a situation where I have time dependent conductances for a river system.
Stages and bottom are fixed in time.

This currently results in somethin like this:
```
[riv]
    stage_p? = stage.idf
    cond_p2 = cond_20010101.idf
```

Stage will be set for all stress periods, or not? And will it crash, since no cond_p1 has been defined?
At any rate, I think it would be better to write in the runfile:

```
[riv]
    stage_p2:N = stage.idf
    cond_p2 = cond_20010101.idf
    cond_p3 = etc.
```

Where `N` is the number of stress periods. Seems minimally ambiguous.

The alternative is backward filling, but doesn't feel right (since forward fill is the default).

Also, I'm not quite sure what iMOD-wq will take, and what it won't!

**Comment by Manangka** [comment]
In GitLab by @Huite on Aug 24, 2019, 13:22

Turns out imod-wq crashes when e.g. a well isn't present in the first stress period, but it is in the second.
(Makes sense, since it allocates at the start of the simulation.)

So an error should be raised if the input of a package starts at a later point than the simulation.

**Comment by Manangka** [comment]
In GitLab by @Huite on Sep 3, 2019, 14:59

Addressed in da38fc5a3e8beb95caa296783c748ed7647076fb

**Comment by Manangka** [comment]
In GitLab by @Huite on Sep 3, 2019, 14:59

closed
